### ISSUE TITLE
Crash to Desktop

### ISSUE THREAD

**Issue by DragonDildos6669** [anegostudios/VintageStory-Issues#192]
I was running along-side a Friend In Mulitplayer to help them get their items back, I jumped up a small Bluff, Crashed.

```
Running on 64 bit Windows with 16GB RAM 
9/26/2018 3:37:58 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at _ycDrqDMbyJhxP01CVRZiivaIisy._BjOzBr8L7yT2ftcm1lkbhtb3TJX(Block , Block , Int32 , Int32 , Int32 , IMapChunk )
   at _A0hpt10KzCwnjqfVcZmZ2lmXlii._ervq6XzpcGNdUaDMX0JxmjQD1Lc(UInt16 , BlockPos , ItemStack )
   at _qcMN8tuA1EmZ2xv2hJLHJFXdyXI._IYvT3yrDg1NRs5bMBEZTNJg2GOb(_YExvbdvkAEr1ujCAPUf21wELkqF )
   at _SV0E94sNjgKgPxI4roqhbEcmpkJ._WBfXs64FuxqYcY5beD1hJlos2u3(_DiufGW68SvMbVaAsZfQM0Hxy5Jo , Single )
   at _DiufGW68SvMbVaAsZfQM0Hxy5Jo._eUWaFj15RslvpUWRF3qvvvvvqxm(Single )
   at _F6pmAk97nl4Y68JTm75LKfNNCP._Fngu1UOZr0sF9zHYLJ7U3ingbSe(Single )
   at _ScUIJvJBZpF2UD0fYUyYgdwbew._naH8bYQMIMCP2t3vfZ9gjnvXAgA(Single )
   at _ScUIJvJBZpF2UD0fYUyYgdwbew._qxTElk8iKgbFFaem3JCd1UwE37e(Single )
   at _wBzvKPrX9UL1M27bH52vbRfijjk._trnerFIACl7sCOEHv2g663E6bmu(Object , FrameEventArgs )
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _LkuRhaQi2csuYkiSJ7OawRUtl5B._SwgdyeohaAdZwoTSgjLzyRUaEXC(_qxKBIIYHRbIS2ztIWz44Pw8ydcP , String[] )
   at _HL8HdS33SaPAV3PFtWs6H3ikCVW._SwgdyeohaAdZwoTSgjLzyRUaEXC(ThreadStart )
```

**Comment by DragonDildos6669** [comment]
[client-debug.txt](https://github.com/anegostudios/VintageStory-Issues/files/2421522/client-debug.txt)
[client-main.txt](https://github.com/anegostudios/VintageStory-Issues/files/2421523/client-main.txt)
[VintagestoryCrash.txt](https://github.com/anegostudios/VintageStory-Issues/files/2421524/VintagestoryCrash.txt)
[client-chat.txt](https://github.com/anegostudios/VintageStory-Issues/files/2421525/client-chat.txt)

**Comment by DragonDildos6669** [comment]
Thank you, have a good day.

**Comment by tyronx** [comment]
Thanks for the report, I thiiiink i fixed this in v1.7 which should become stable very soon!
